fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/election/adapters/ElectionSetupViewPagerAdapter.java

```diff
@@ -184,6 +196,18 @@ public List<Integer> getValidInputs() {
     return listOfValidInputs;
   }
 
+  /**
+   * This function checks if a given list of strings has at least a duplicate. The comparison is
+   * performed case-insensitive.
+   *
+   * @param strings List of Strings
+   * @return true if the list of strings has at least a duplicate, false otherwise
+   */
+  public static boolean hasDuplicate(List<String> strings) {
+    Set<String> uniqueStrings = new HashSet<>();
+    return strings.stream().map(String::toLowerCase).anyMatch(s -> !uniqueStrings.add(s));
```

Comment:
A case insensitive comparison makes sense from a usability perspective, but you might want to sync up with fe1 to make sure that you have the same definition of a valid input :)

Reply:
Right, going to change that!